packages/styles/scss/patterns/sub-patterns/content-section/_content-section.scss

```diff
@@ -0,0 +1,64 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@import '../../../globals/imports';
+
+@mixin themed-items {
+  color: $text-01;
+  background: $ui-background;
+}
+
+@mixin content-section {
+  .#{$prefix}--content-section {
+    @include carbon--make-row;
+
+    &__heading {
+      margin-bottom: $carbon--layout-03;
+      @include carbon--type-style('expressive-heading-04', true);
+
+      &__col {
+        @include carbon--make-col-ready;
+        @include carbon--make-col(4, 4);
+
+        @include carbon--breakpoint(lg) {
+          @include carbon--make-col(4, 16);
+        }
+      }
+    }
+    &__children {
+      &__col {
+        @include carbon--make-col-ready;
+        @include carbon--make-col(4, 4);
+
+        @include carbon--breakpoint(lg) {
+          @include carbon--make-col(12, 16);
+        }
+      }
+    }
+
+    padding-top: $carbon--layout-05;
+    padding-bottom: $carbon--layout-05;
+
+    @include carbon--breakpoint('md') {
+      padding-bottom: $carbon--layout-06;
+    }
+    @include carbon--breakpoint('lg') {
+      padding-top: $carbon--layout-05;
```

Comment:
According to spec I'm seeing a `48px` here instead of `32px`. So that would be a `layout-04` I believe.

![image](https://user-images.githubusercontent.com/3793636/74487180-72639580-4e84-11ea-9174-1a5ae4c91fe8.png)

Reply:
Taking the heading margin into account this would become `layout-03` instead.